### PR TITLE
Audiorecording share edit options

### PR DESCRIPTION
**Pull request by JB184351**

### CODE REVIEW

File: TalkAloud/TalkAloud/Cells/MoreOptionsTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//
+//  Created by Justin Bengtson on 8/22/20.
+//  Copyright © 2020 Justin Bengtson. All rights reserved.
+//
+
+import UIKit
+
+class MoreOptionsTableViewCell: UITableViewCell {
+
+    override func awakeFromNib() {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Cells/MoreOptionsTableViewCell.swift]: Please remove this boilerplate code

File: TalkAloud/TalkAloud/Cells/MoreOptionsTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Cells/MoreOptionsTableViewCell.swift]: Please remove this boilerplate code

File: TalkAloud/TalkAloud/Managers/CoreDataManager.swift [hunk 3]
```diff
@@ -99,6 +125,33 @@ class CoreDataManager {
...
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+
+        let managedContext = appDelegate.persistentContainer.viewContext
+        let changeRequest = NSFetchRequest<NSFetchRequestResult>(entityName: "AudioRecordingObject")
+
+        do {
+            let objects = try managedContext.fetch(changeRequest)
+            
+            for object in objects {
+                let recordingInObjects = AudioRecording(object: object as! NSManagedObject)
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/Managers/CoreDataManager.swift]: Why not the same variable name like in the others?

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift [hunk 4]
```diff
@@ -34,36 +49,65 @@ class AudioRecordingsTableViewController: UITableViewController {
...
     
     
-    func filter(by tags: [String]) {
-        isFiltered = true
-        filteredAudioRecordings = AudioManager.sharedInstance.filteredAudioRecordings(with: tags)
-        tableView.reloadData()
+}
+
+extension AudioRecordingsViewController: TagFilterDelegate {
+    func didUpdateTagToFilter(by tags: [String]?) {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift:55]: Add an empty line

File: TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift [hunk 5]
```diff
@@ -34,36 +49,65 @@ class AudioRecordingsTableViewController: UITableViewController {
...
+            filter(by: tags)
+        } else {
+            isFiltered = false
+            recordingsTableView.reloadData()
+        }
+    }
+}
+
+extension AudioRecordingsViewController: AudioRecordingCellDelegate {
+    func didTappedMoreButton(for cell: AudioRecordingCell) {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift:67]: Add an empty line

File: TalkAloud/TalkAloud/ViewControllers/MoreOptionsViewController.swift [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+        tagAlertController.addAction(removeTagAction)
+        tagAlertController.addAction(cancelTagAction)
+
+        self.present(tagAlertController, animated: true)
+    }
+    
+}
+
+extension MoreOptionsViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/MoreOptionsViewController.swift:116]: Add an empty line

File: TalkAloud/TalkAloud/ViewControllers/MoreOptionsViewController.swift [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        cell.textLabel?.text = moreOptions[indexPath.row].title
+        
+        return cell
+    }
+    
+    
+}
+
+extension MoreOptionsViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```
Comment (virgil-adumitroae) [inline TalkAloud/TalkAloud/ViewControllers/MoreOptionsViewController.swift:133]: Add an empty line

**Review by virgil-adumitroae — CHANGES_REQUESTED** [review]

File: TalkAloud/TalkAloud/Managers/CoreDataManager.swift (hunk 3 above)
Comment (JB184351) [reply TalkAloud/TalkAloud/Managers/CoreDataManager.swift]: I thought I fixed that 😅

**Pull request merged**